### DISCUSSION TITLE
Can yall hack Harmony Island Arena that coming to prodigy very soon?

### DISCUSSION THREAD

**Discussion by Protuber** [Prodigy-Hacking/ProdigyMathGameHacking#975] [(A) Anything]
This is a question and I want to know if yall can hack the items on harmony and arena coming soon

**Comment by ghost** [comment]
oooh good idea

**Reply by Protuber** [reply]
thanks

**Reply by ghost** [reply]
yeah i realy want that gold armour

**Reply by Protuber** [reply]
same

**Reply by Protuber** [reply]
when it come try it  it it works  im so to be happy because I really want that gold armour looks sick but I swear to god if the gold armour only have as  a heart bonus of 6 I'm going to lose it

**Reply by ghost** [reply]
@Protuber what heartbonus is it

**Reply by ghost** [reply]
how come the rainbow armour and silvel and gold are not included in cheat menu my friend dose not hack and he has the rainbow hat

**Reply by ghost** [reply]
allso i played in harmony island arena yesterday

**Reply by ghost** [reply]
but then i reinstalled the cheat menu and arena was the ld one again

**Reply by ghost** [reply]
@Protuber

**Reply by ghost** [reply]
@Protuber i forgot

**Reply by ghost** [reply]
allso script based hacks are back

**Reply by ghost** [reply]
no

**Comment by ghost** [comment]
thanks to ceat menu my player looks like this 
![image](https://user-images.githubusercontent.com/83719352/120025040-f0c64700-bfa4-11eb-9809-3e0e98d49b12.png)

**Reply by ghost** [reply]
lol no aincient relic

**Reply by ghost** [reply]
*not him*

**Reply by ghost** [reply]
*bonjour* @Protuber @BETPRODIGYHACKER  *myself*
